README.md

```diff
@@ -1,87 +1,197 @@
-# Name of project
+# Job Advisor
 
-> A longer description
+> Job Advisor helps organisations promote right values and job seekers to make
+> right choices for their career.
 
 ## Table of contents
 
-- [Name of project](#name-of-project)
+- [Name of project](#job-advisor)
   - [Table of contents](#table-of-contents)
   - [General info](#general-info)
-  - [Screenshots](#screenshots)
-  - [Technologies](#technologies)
-  - [Setup](#setup)
-  - [Code Examples](#code-examples)
-  - [Features](#features)
-  - [Status](#status)
+  - [Design protoype](#design-protoype)
+  - [Tools and Technologies](#tools-and-technologies)
+    - [UX/UI](#uxui)
+    - [Frontend](#frontend)
+    - [Backend](#backend)
+    - [Devops](#devops)
+  - [Sample React code](#sample-react-code)
+  - [App demo video](#app-demo-video)
   - [Inspiration](#inspiration)
   - [Contact](#contact)
   - [Instructions for use](#instructions-for-use)
   - [Code Quality Checks](#code-quality-checks)
-  - [Continuous Integration (CI)](#continuous-integration-ci)
-  - [Repo Setup](#repo-setup)
+  - [Authors](#authors)
 
 ## General info
 
-> A short description
+> At Job Advisor, We help people to get jobs. We believe financialindependence
```

Comment:
Space is needed in financial independence.

Reply:
space has given